extension/tasks/dependabotV2/index.ts

```diff
@@ -111,25 +111,27 @@ async function run() {
         failedJobs++;
       }
 
-      // Run an update job for each existing pull request; this will resolve merge conflicts and close pull requests that are no longer needed
-      if (!taskInputs.skipPullRequests) {
-        for (const pullRequestId in existingPullRequests) {
-          const updatePullRequestJob = DependabotJobBuilder.newUpdatePullRequestJob(
-            taskInputs,
-            pullRequestId,
-            update,
-            dependabotConfig.registries,
-            existingPullRequestDependencies,
-            existingPullRequests[pullRequestId],
-          );
-          const updatePullRequestOutputs = await dependabot.update(updatePullRequestJob, dependabotUpdaterOptions);
-          if (!updatePullRequestOutputs || updatePullRequestOutputs.filter((u) => !u.success).length > 0) {
-            updatePullRequestOutputs.filter((u) => !u.success).forEach((u) => exception(u.error));
-            failedJobs++;
+      // If there are existing pull requests, run an update job for each one; this will resolve merge conflicts and close pull requests that are no longer needed
+      if (existingPullRequests && existingPullRequests.keys.length > 0) {
```

Comment:
@mburumaxwell sorry, I had a brain fart here, this line should be:

```
if (existingPullRequests && Object.keys(existingPullRequests)?.length > 0) {
```
Are you able to edit in main? Otherwise I will make a PR can update later today.
I really need to get started on those unit tests :sweat:

Reply:
Haven't heard "brain fart" before 😸 
https://github.com/tinglesoftware/dependabot-azure-devops/commit/7b0ed03a1082e8de273c8ae7bfb1f0c16217401e